### PR TITLE
Bump version and add trancelove as developer

### PR DESCRIPTION
**Pull request by VishalNehra**
v3.3.0RC1 release bump as discussed in #1315

### CODE REVIEW

**Comment by EmmanuelMess** [comment]
Not to master, to release-3.3.0

File: app/src/main/java/com/amaze/filemanager/activities/AboutActivity.java [hunk 1]
```diff
@@ -46,15 +46,17 @@
...
 
     private static final String KEY_PREF_STUDIO = "studio";
     private static final String URL_AUTHOR_1_G_PLUS = "https://plus.google.com/u/0/110424067388738907251/";
     private static final String URL_AUTHOR_1_PAYPAL = "arpitkh96@gmail.com";
     private static final String URL_AUTHOR_2_G_PLUS = "https://plus.google.com/+VishalNehra/";
     private static final String URL_AUTHOR_2_PAYPAL = "https://www.paypal.me/vishalnehra";
     private static final String URL_DEVELOPER1_GITHUB = "https://github.com/EmmanuelMess";
+    private static final String URL_DEVELOPER2_GITHUB = "https://github.com/TranceLove";
     private static final String URL_DEVELOPER1_BITCOIN = "bitcoin:12SRnoDQvDD8aoCy1SVSn6KSdhQFvRf955?amount=0.0005";
+    private static final String URL_DEVELOPER2_DONATE = "https://www.paypal.me/raymondlai";
```
Comment (TranceLove) [inline app/src/main/java/com/amaze/filemanager/activities/AboutActivity.java]: Please change to https://paypal.me/TranceLove - I've just set things up there, and I'm using the same profile pic as here.

**Comment by VishalNehra** [comment]
It is 287 commits behind master.

File: app/src/main/java/com/amaze/filemanager/activities/AboutActivity.java (hunk 1 above)
Comment (VishalNehra) [reply app/src/main/java/com/amaze/filemanager/activities/AboutActivity.java]: Done

**Comment by EmmanuelMess** [comment]
> It is 287 commits behind master.

My mistake

**Comment by VishalNehra** [comment]
Why a seperate branch then? I never noticed it.

**Comment by EmmanuelMess** [comment]
> Why a seperate branch then? I never noticed it.

Features aren't added to 3.3.0, fixes are cherry-picked to release-3.3.0.

**Comment by VishalNehra** [comment]
I understand. So we can safely build master and delete release-3.3.0 then?

**Comment by VishalNehra** [comment]
Because it doesn't have all the work.

**Comment by EmmanuelMess** [comment]
> Because it doesn't have all the work.

That's the whole idea of the matter, not all the new things are in the branch thus less bugs.

> I never cherry picked nor did I made any PR to release-3.3.0 branch. Wasn't aware of it.

I cherry picked, now you are aware, do check the branch for mistakes.

**Pull request merged**